tools/docker/corda-all-in-one/README.md

```diff
@@ -22,6 +22,28 @@ docker run --rm --privileged caio
 ### Build and Run Image Locally
 
 ```sh
-docker build ./tools/docker/corda-all-in-one/ -f ./tools/docker/corda-all-in-one/corda-v4_8/Dockerfile -t caio48
+DOCKER_BUILDKIT=1 docker build ./tools/docker/corda-all-in-one/ -f ./tools/docker/corda-all-in-one/corda-v4_8/Dockerfile -t caio48
 docker run --rm --privileged caio48
+```
+
+# cactus-corda-4-8-all-in-one-flowdb
+
+> This docker image is for `testing` and `development` only.
+> Do NOT use in production!
+
+## Customization
+
+`build.gradle` file from this sample has defined a single node called PartyA. It was modified to deploy the same nodes as in the obligation sample to make it work with our CordaTestLedger:
```

Comment:
@petermetz I added this regarding your comment https://github.com/hyperledger/cactus/pull/1561#discussion_r762627108

Reply:
@elenaizaguirre Great, thank you very much!